clip_image.py

```diff
@@ -81,7 +81,7 @@ def _create_embeddings(self, document_batches_generator: Iterable):
             for document_batch in document_batches_generator:
                 blob_batch = [d.blob for d in document_batch]
                 if self.use_default_preprocessing:
-                    images = [Image.fromarray(blob) for blob in blob_batch]
+                    images = [Image.fromarray(blob.astype(np.uint8)) for blob in blob_batch]
```

Comment:
Note that this conversion might go wrong without throwing any error. Since we have mentioned this in the README, I suggest not doing this. 
```suggestion
                    images = [Image.fromarray(blob.astype(np.uint8)) for blob in blob_batch]
```